fares-data-build-tool/create-data
# feat: disabling any usage of TNDS data

## Description

TNDS data can currently be used, but no Netex gets generated. This change disables usage of TNDS data, until a time we delete it entirely or reinstate it.

## Testing instructions

Navigate to /service page and /serviceList pages to check you cannot switch to TNDS.
